docs/connectors/slack.md

```diff
@@ -91,27 +92,26 @@ on using slack in Socket Mode.
 * On the left column go to "Event Subscriptions" and set the "Enable Events" toggle to enabled.
 * Under "Request URL" add the `/connector/slack` uri to your endpoint: https://slackbot.example.com/connector/slack. Note that you will have to have your Opsdroid instance running so Slack can verify the endpoint.
 
-**Socket Mode**
-
- Mode. The reason for this is that the Request URL verification step is needed and is only available via the *Events API*.
+**2. Socket Mode**
 * Go to your [Slack App](https://api.slack.com/apps)
-* On the left column go to "Socket Mode" and set the "Enable Socket Mode" toggle to enabled.
+* On the left column go to "Socket Mode", set the "Enable Socket Mode" toggle to enabled and give your token a name.
+* Copy your token add add it to your Opsdroid configuration file as your `app-token`. (the `app-token` will start with `xapp-abcdef-1233`)
 * Make sure to set `socket-mode` to `true` in your Opsdroid configuration.
-* Copy your new token add add it to your Opsdroid configuration file as your `app-token`. (the `app-token` will start with `xapp-abcdef-1233`)
 
 
 (subscribe-to-events)=
 ### Subscribe to events
 You will need to subscribe to events in your new Slack App, so Opsdroid can receive those events.
 You need to subscribe to events regardless of the backend: **Socket Mode** or **Events API**
+* On the left column go to "Event Subscriptions" and set the "Enable Events" toggle to enabled.
 * Under "Subscribe to bot events" choose the events you want to subscribe for. You need at least one, `message.channels` will allow you to receive events everytime a message is posted into a channel. The following events are also supported by opsdroid: `message.im`, `channel_archive`, `channel_unarchive`, `channel_created`, `channel_rename`, `pin_added`, `pin_removed` and `team_join`.
-* Don't forget to save your changes in the slack app.
-(usage)=
```

Comment:
I'm afraid it does look like you've removed this reference

Reply:
oh! ok, I will make the changes. Thank you for the feedback!